demos/foodie/index.html

```diff
@@ -47,7 +47,7 @@ <h1>[count(dish)] Notable Dish(es)</h1>
 						<p property="name">Filet mignon with black truffle and foie gras</p>
 						<p>
 							Rating:
-							<meter property="dishRating" min="0" value="4" max="5" mv-editor-step="0.5"></meter>
+							<meter property="dishRating" min="0" value="4" max="5" mv-edit-step="0.5"></meter>
```

Comment:
We should release a new version that includes this. There was a long period without commits, that would be a good candidate.

Reply:
Totally agree!